Apps documentation reconstruction branch (#1226)
YunoHost/doc: app_baikal.md

# <img src="/images/baikal_logo.png" height="80px" alt="Baïkal's logo"> Baïkal

[![Install Baïkal with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=baikal) [![Integration level](https://dash.yunohost.org/integration/baikal.svg)](https://dash.yunohost.org/appci/app/baikal)

- [Configuration](#Configuration)
  - [Login to admin interface](#Login-to-administration-interface)
  - [Creating a new user](#Create-a-new-user)
- [CalDAV connection](#CalDAV-connection)
  - [Connecting Thunderbird with Lightning](#Connecting-Thunderbird-with-Lightning)
  - [Connecting to AgenDAV](#Connecting-to-AgenDAV)
- [CardDAV Connection](#CardDAV-Connection)
- [Useful links](#Useful-links)

> Baïkal is a server for calendars and address book, which used CalDav and CardDav protocol. Baïkal can be synced with a lot of client like Thunderbird + Lightning.

Reviewer: ```suggestion
Baïkal is a server for calendars and address books, which uses the CalDav and CardDav protocol. Baïkal can be synced with a lot of clients, like Thunderbird + Lightning.
```

**WARNING**: Baikal will not work if you have installed a **Nextcloud** ( their cardav/caldav functions conflict).

## Configuration

### Connecting to the administration interface

> To configure the application, go to the address: `sub.domain.tld/admin` or `domain.tld/baikal/admin`.

Reviewer: ```suggestion
To configure the app, go to the address: `sub.domain.tld/admin` or `domain.tld/baikal/admin`.
```

> The username to specify is `admin` followed by the specific password you chose when installing Baikal. Be careful, the password must not contain any special characters.

Reviewer: ```suggestion
The username to specify is `admin`, followed by the specific password you chose when installing Baïkal. The password can contain any special characters.
```
